packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1103,6 +1104,45 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// Override in subclasses with children and call the visitor for each child.
   void visitChildren(RenderObjectVisitor visitor) { }
 
+  /// Capture an image of the current state of the render object and its
+  /// children.
+  ///
+  /// The returned [ui.Image] has uncompressed raw bytes in the dimensions of
+  /// the render object, multiplied by the [window.devicePixelRatio].
```

Comment:
Don't say "multiplied by the [window.devicePixelRatio]" since it's covered below in the discussion of the [pixelRatio] named parameter?